### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by jesec**

### CODE REVIEW

File: client/src/javascript/i18n/translations/zh-Hant.json [hunk 1]
```diff
@@ -264,7 +264,7 @@
   "torrents.list.no.torrents": "沒有任何種子可以顯示。",
   "torrents.list.drop": "將檔案拖曳到這裡來新增到 rTorrent。",
   "torrents.list.cannot.connect": "無法連線到 rTorrent。",
-  "torrent.list.peers": "{total} 中的 {connected}",
+  "torrent.list.peers": "{total} / {connected}",
```
Comment (vongola12324) [inline client/src/javascript/i18n/translations/zh-Hant.json:267]: Outdated.

Comment (vongola12324) [reply client/src/javascript/i18n/translations/zh-Hant.json:267]: There is a new translation in Crowdin.

Comment (jesec) [reply client/src/javascript/i18n/translations/zh-Hant.json:267]: Crowdin will push updates automatically after a while.

Comment (jesec) [reply client/src/javascript/i18n/translations/zh-Hant.json:267]: Ack. I will wait for the updates before merging.

**Pull request merged**